parser.go

```diff
@@ -1588,13 +1588,52 @@ func (p *Parser) tryParseNamedArg() *ast.NamedArg {
 	}
 }
 
+func (p *Parser) lookaheadLambdaArg() bool {
+	lexer := p.Lexer.Clone()
+	defer func() {
+		p.Lexer = lexer
+	}()
+
+	if p.Token.Kind != "(" && p.Token.Kind != token.TokenIdent {
+		return false
+	}
+
+	// Note: all lambda patterns can be parsed as expr -> expr.
+	p.parseExpr()
+	return p.Token.Kind == "->"
+}
+
+func (p *Parser) parseLambdaArg() *ast.LambdaArg {
+	lparen := token.InvalidPos
+	var args []*ast.Ident
+	if p.Token.Kind == "(" {
+		lparen = p.expect("(").Pos
+		args = parseCommaSeparatedList(p, p.parseIdent)
+		p.expect(")")
+	} else {
+		args = parseCommaSeparatedList(p, p.parseIdent)
```

Comment:
I think this line should be `[]*ast.Ident{p.parseIdent()}` because multiple argument names without parentheses (e.g., `a, b -> ...`) are already rejected by `lookaheadLambdaArg`.

Reply:
FIxed in https://github.com/cloudspannerecosystem/memefish/pull/219/commits/abdf190136162327d10696ce94cd83900d174e96